make_grammar.js

```diff
@@ -96,6 +98,8 @@ module.exports = function make_grammar(dialect) {
                         $._pipeline_action,
                         $.if_action,
                         $.range_action,
+                        $.break_action,
+                        $.continue_action,
```

Comment:
This would allow

template: main:10: {{continue}} outside {{range}}

But idk if we should enforce this.